.github/workflows/eslint.yml

```diff
@@ -2,12 +2,17 @@ name: ESLint
 
 on:
   workflow_call:
+  schedule:
+    - cron: '0 7 * * 1'
```

Comment:
How does this work, will it still run the job on each PR update or only on the weekly schedule now? 🤔

Reply:
It will run on all PR, and push events (like if you go crazy and decide to push to master 😅) via the workflow call + on a weekly schedule. 

The weekly schedule is recommended by CodeQL as there could be longs periods without pushes or PRs to master and keeping it on a schedule will ensure it identifies new issues (when the CodeQL rules update) even if there is no activity to master.